src/components/Camera/LivenessCamera.js

```diff
@@ -19,17 +20,20 @@ type Props = {
 
 type State = {
   currentIndex: number,
-  hasTimedOut: boolean,
   isRecording: boolean,
+  hasBecomeInactive: boolean,
+  hasRecordingTakenTooLong: boolean,
 }
 
 const initialState = {
   currentIndex: 0,
   isRecording: false,
-  hasTimedOut: false,
+  hasBecomeInactive: false,
+  hasRecordingTakenTooLong: false,
 }
 
-const livenessTimeout = { name: 'LIVENESS_TIMEOUT', type: 'warning' }
+const inactiveError = { name: 'CAMERA_INACTIVE', type: 'warning' }
```

Comment:
Minor: do we have any informal convention for `const` values? like uppercase or something?

Reply:
Only for Redux action types, but not for any others